Source/Urho3D/Graphics/OpenGL/OGLGraphics.cpp

```diff
@@ -53,6 +53,12 @@
 #endif
 
 #ifdef __EMSCRIPTEN__
+#include "../Input/Input.h"

```

Comment:
These relative includes are incorrect and work only by chance. It should be `../../Input/Input.h`.